docs/workflow/requirements/linux-requirements.md

```diff
@@ -48,13 +48,15 @@ The packages you need to install are shown in the following list:
 - `libssl-dev`
 - `libkrb5-dev`
 - `ninja-build` (Optional. Enables building native code using `ninja` instead of `make`)
+- `pigz` (Optional. Enables parallel gzip compression for tarball creation in `packs` subset)
+- `cpio` (Optional. Needed to build RPM packages)
```

Comment:
I'm not sure I'd list it as optional since we build rpm by default now after https://github.com/dotnet/runtime/pull/109526 and the build will fail if it's not there.

Reply:
Good point. I'll remove the note and add it with rest of the required packages.